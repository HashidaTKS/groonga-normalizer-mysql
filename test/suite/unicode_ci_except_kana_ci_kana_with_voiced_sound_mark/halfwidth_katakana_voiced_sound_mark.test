register normalizers/mysql

normalize NormalizerMySQLUnicodeCIExceptKanaCIKanaWithVoicedSoundMark \
  "beforeｹﾞafter"
