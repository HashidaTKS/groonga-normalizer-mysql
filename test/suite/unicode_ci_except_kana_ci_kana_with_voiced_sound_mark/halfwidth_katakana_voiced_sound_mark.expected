register normalizers/mysql
[[0,0.0,0.0],true]
normalize NormalizerMySQLUnicodeCIExceptKanaCIKanaWithVoicedSoundMark   "beforeｹﾞafter"
[[0,0.0,0.0],{"normalized":"BEFOREけAFTER","types":[]}]
